core/che-core-metrics-core/pom.xml

```diff
@@ -33,6 +33,12 @@
         <dependency>
             <groupId>com.squareup.okhttp3</groupId>
             <artifactId>okhttp</artifactId>
+            <exclusions>
+                <exclusion>
+                    <artifactId>okio</artifactId>
+                    <groupId>com.squareup.okio</groupId>
+                </exclusion>
+            </exclusions>
         </dependency>
```

Comment:
Do you use OkHttp for something else than the client?
If not, you should probably remove the entire dependency definition

Reply:
Yes, we use it in the `OkHttpMetricsEventListenerProvider` https://github.com/eclipse-che/che-server/blob/6670c6986b1cb3da5e733e1a0b48db78b393ee3d/core/che-core-metrics-core/src/main/java/org/eclipse/che/core/metrics/OkHttpMetricsEventListenerProvider.java#L26

Reply:
Sounds good, but if these metrics are related in some way to the Fabric8 client, then it needs to be reimplemented. Or switch to use OkHttp as the underlying HttpClient for Fabric8.

Reply:
Since we do not have any subscriptions to  the OkHTTP3 events, I removed the provider and the dependency.